student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'Unknow', parent_permission: false)
+    super(age, name, parent_permission)
```

Comment:
- [ ] The `parent_permission` parameter uses the colon notation so that means we need to assign it correctly here to avoid ArgumentError.

  ```
  super(age, name, parent_permission: parent_permission)
  ```